packages/wallets/src/features/cfd/screens/ChangePassword/ChangePassword.scss

```diff
@@ -0,0 +1,73 @@
+.wallets-change-password {
+    &__modal-wrapper {
+        width: 90.4rem;
+        height: 68.8rem;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+
+        @include mobile {
+            width: auto;
+            padding: 1.6rem;
+        }
+    }
+
+    &__container {
+        margin: 0 auto;
+        padding-top: 2.4rem;
+        width: 45.2rem;
+        height: 100%;
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        @include mobile {
+            width: 100%;
+        }
+    }
+
+    &__tab {
+        padding-top: 2.4rem;
+        margin-bottom: 3.2rem;
+        justify-self: center;
+        width: 100%;
+        &--btn {
+            width: 50%;
+            border: none;
+            border-bottom: 2px solid var(--light-7-secondary-background, #f2f3f4);
+            background: none;
+            padding: 1rem 0;
+            cursor: pointer;
+        }
+        &--btn-active {
+            width: 50%;
+            border: none;
+            border-bottom: 2px solid var(--brand-coral, #ff444f);
+            background: none;
+            padding: 1rem 0;
+        }
+    }
+
+    &__content {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        gap: 2.4rem;
+        &--icon {
+            text-align: center;
+        }
+        &--text {
```

Comment:
same as previous comment - thats block, --icon and --btn too, 

more elaboration on the same topic from different PR
https://github.com/binary-com/deriv-app/pull/11434#discussion_r1393871446